### ISSUE TITLE
[new-rule] prefer-to-be

### ISSUE THREAD

**Issue by LitoMore** [jest-community/eslint-plugin-jest#857]
`toBe` is simpler than `toEqual`. Let's use `toBe` instead of `toEqual` for those values can use `Object.is` directly.

The following pattern is considered warning:

```js
expect(foo).toEqual(false);
expect(foo).toEqual(1);
expect(foo).toEqual('text');
```

The following pattern is not warning:

```js
expect(foo).toBe(false);
expect(foo).toBe(1);
expect(foo).toBe('text');
```

What do you think about this rule?

I would like to help if you want this.

**Comment by G-Rath** [comment]
We've actually already got an issue for this feature + a PR, I've just not yet had the time to merge it :)

**Comment by LitoMore** [comment]
Cool!
